laa_court_data_api_app/routers/defendants.py

```diff
@@ -0,0 +1,65 @@
+import logging
+from typing import Optional
+from uuid import UUID
+
+from fastapi import APIRouter
+from fastapi.responses import Response
+
+from laa_court_data_api_app.internal.court_data_adaptor_client import CourtDataAdaptorClient
+from laa_court_data_api_app.models.defendants.defendants_response import DefendantsResponse
+from laa_court_data_api_app.models.prosecution_cases.prosecution_cases_results import ProsecutionCasesResults
+
+logger = logging.getLogger(__name__)
+
+router = APIRouter()
+
+
+@router.get('/v2/defendants', response_model=DefendantsResponse, status_code=200)
+async def get_defendants(urn: Optional[str] = None,
+                         name: Optional[str] = None,
+                         dob: Optional[str] = None,
+                         uuid: Optional[UUID] = None):
+    client = CourtDataAdaptorClient()
+    logger.info("Calling GET Endpoint")
```

Comment:
Can we be more specific with the log message here please

Reply:
This logs as: 
`logger=laa_court_data_api_app.routers.defendants get_defendants() Calling GET Endpoint`

Each of the paths in the conditionals have more specific details of what is called. At this point, the route was called but not yet matched, what else would be useful here?

Reply:
If you were to try and search for this specific message in a list of logs it might be difficult. Something that follows the style currently would be `Calling_Defendants_Get_Endpoint`

Reply:
Make sense - I'll also add the same to hearing summaries